packages/web-config-server/src/apiV1/dataBuilders/generic/analytics/analytics.js

```diff
@@ -13,9 +13,10 @@ class AnalyticsBuilder extends DataBuilder {
   }
 }
 
-export const analytics = ({ dataBuilderConfig, query, entity }, aggregator, dhisApi) => {
+export const analytics = ({ models, dataBuilderConfig, query, entity }, aggregator, dhisApi) => {
```

Comment:
The next few files are data builders that were added since I last worked on this branch, and so need the same changes to add `models` as the other data builders that were already reviewed

Reply:
I see... Unfortunately we'll have to keep an eye on those until the feature branch is merged!

Refactoring to use a single point of data builder instantiation seems the way to go (hooray for the reporting package)!